aiohttp/web_protocol.py

```diff
@@ -105,7 +104,6 @@ async def log(
         self.access_logger.log(request, response, self._loop.time() - request_start)
 
 
-@attr.s(auto_attribs=True, frozen=True, slots=True)
```

Comment:
As per the linter failure, this can't just be removed. The functionality would need to be replaced.
I'd recommend taking a look at what was done in the original PR linked in the issue:
https://github.com/aio-libs/aiohttp/pull/5284/files